app/retail/templates/emails/grants/transaction_summary.html

```diff
@@ -0,0 +1,224 @@
+ it under the terms of the GNU Affero General Public License as published
+ by the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with this program. If not, see <http://www.gnu.org/licenses/>. 
+{% endcomment %}
+{% load i18n humanize static %} 
+
+{% block content %}
+
+<style>
+  hr {
+    width: 80%;
+    height: 2px;
+    border-radius: 25px;
+    border: none;
+    background-color: #D2D2D2;
+  }
+
+  .grantInfo {
+    padding-top: 30px;
+    padding-bottom: 1%;
+  }
+
+  .grantInfo a img {
+    border-radius: 25px;
+    margin-bottom: 10px;
+  }
+
+  .grantInfoName {
+    font-size: 20px;
+    color: #0D0764;
+  }
+
+  .grantInfo h5 {
+    font-size: 14px;
+    margin-top: 15px;
+    margin-top: 30px;
+  }
+
+  .grantInfo h3 {
+    margin-top: -20px;
+  }
+
+  .updatesButton {
+    padding-left: 30px;
+    padding-right: 30px;
+    text-decoration: none;
+    padding-top: 8px;
+    padding-bottom: 8px;
+    background-color: #0D0764;
+    color: white;
+    border-radius: 2.7px;
+    font-size: 17px;
+  }
+
+  #grow-oss {
+    margin-top: 70px;
+    margin-bottom: 40px;
+  }
+
+  .progress {
+    width: 60%;
+    margin-left: 20%;
+    margin-top: 50px;
+    padding-bottom: 75px;
+  }
+
+  #tablePadding {
+    padding-bottom: 25px;
+  }
+
+  .progressBar {
+    height: 15px;
+    background-color: #D8D8D8;
+    border-radius: 2.7px;
+  }
+
+  .progressCompleted {
+    height: 100%;
+    background-color: #0FCE7C;
+    border-radius: 2.7px;
+    float: left;
+  }
+
+  .progressRemoved {
+    height: 100%;
+    background-color: #CF0E0E;
+    border-top-right-radius: 2.7px;
+    border-bottom-right-radius: 2.7px;
+    float: left;
+  }
+
+  .funding {
+    width: 100%;
+  }
+
+  .leftFunding {
+    width: 50%;
+    float: left;
+    text-align: left;
+  }
+  
+  .rightFunding {
+    width: 50%;
+    float: left;
+    text-align: right;
+  }
+
+  .funding>div>h4 {
+    line-height: 0px;
+    color: #0D0764;
+    padding-top: -50px;
+  }
+
+  .leftFunding>h4 {
+    text-align: left;
+  }
+
+  .rightFunding>h4 {
+    text-align: right;
+  }
+
+  .progressBarLine {
+    width: 1px;
+    height: 45px;
+    background-color: #979797;
+    float: left;
+    transform: translateY(-20%);
+    margin-right: 20px;
+  }
+
+  .funding>div>p {
+    line-height: 0px;
+    color: #BBBDBF;
+  }
+
+  table {
+    width: 100%;
+  }
+
+  td:nth-child(1)>img {
+    border-radius: 25px;
+  }
+
+</style>
+<img id="newSupporter" src="{% static "v2/images/transaction-summary.png" %}" alt="Transaction Summary" title="Transaction Summary">
```

Comment:
Need all copy throughout this file, to include `alt` and `title` values wrapped in `trans`